.github/workflows/release.yml

```
@@ -58,10 +60,20 @@ jobs:
       - uses: actions/download-artifact@v8
         with:
           merge-multiple: true
+      - name: Generate checksums
+        run: sha256sum *.tar.gz *.zip > checksums.txt
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2035:info:1:11: Use ./*glob* or -- *glob* so names with dashes won't become options

(shellcheck)